pkg/time/time.go

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+// package time is a wrapper for the upstream system library for time.Time that
```

Comment:
nit: space between `.` and `Time`

Reply:
Actually this was originally intentional as `time.Time` but the lib ended up wrapping more than just `time.Time` so I'll fix it up to refer that it just wraps the whole upstream library.